src/components/visualizations/chart/highcharts/getOptionalStackingConfiguration.ts

```diff
@@ -141,18 +141,17 @@ export function getParentAttributeConfiguration(chartOptions: IChartOptions, con
 /**
  * Format labels in Y axis from '0 - 100' to '0% - 100%'
  * @param chartOptions
- * @param _
+ * @param _config
  * @param chartConfig
  */
-export function getShowInPercentConfiguration(chartOptions: IChartOptions, _: any, chartConfig: IChartConfig) {
+export function getShowInPercentConfiguration(chartOptions: IChartOptions, _config: any, chartConfig: IChartConfig) {
     const { stackMeasuresToPercent = false } = chartConfig;
     if (!stackMeasuresToPercent) {
         return {};
     }
 
     const { yAxes = [] } = chartOptions;
-    const yAxis = yAxes.map((axis: IAxis) => ({
-            ...axis,
+    const yAxis = yAxes.map(() => ({
             labels: {
                 formatter: partial(formatAsPercent, 1)
```

Comment:
Create a variable for this partial function, for example:
```const percentageFormatter = partial(formatAsPercent, 1)```

instead of creating new function every iteration

Reply:
fixed :)